internal/common/objects/MegaBike.go

```diff
@@ -1,21 +1,20 @@
 package objects
 
 import (
-	//"fmt"
 	utils "SOMAS2023/internal/common/utils"
-	//baseAgent "github.com/MattSScott/basePlatformSOMAS/BaseAgent"
+
 	"github.com/google/uuid"
 )
 
-// MegaBike will have the following forces 
-type MegaBike struct{
-	id uuid.UUID
-	coordinates utils.Coordinates
-	agentForces [] utils.Forces
-	Mass float64
+// MegaBike will have the following forces
+type MegaBike struct {
+	id           uuid.UUID
+	coordinates  utils.Coordinates
+	agentForces  []utils.Forces
+	Mass         float64
 	acceleration float64
-	velocity float64
-	orientation float64
+	velocity     float64
+	orientation  float64
```

Comment:
I think the MegaBike should also contain a map of all agents that are riding it and also implement some function to add/remove agents when they join/ leave the bike + something to retrieve the map